web/src/axios/index.js

```diff
@@ -21,6 +21,8 @@ request.interceptors.request.use(config => {
     if ( user != null && user.name == anonymous) {
         return Promise.reject(anonymous)
     }
+    const token=localStorage.getItem('token')
+    token?config.headers.Authorization=token:null;
```

Comment:
The token has be set from the server side, so it's not necessary set it again.

Reply:
If the client does not set a token at the time of the request, how does the server get it? if do not set it, Authorization is nill.
`func getTokenFromAuthorizationHeader(c *gin.Context) (string, error) {
	auth := c.Request.Header.Get("Authorization")
	if auth == "" {
		return "", nil
	}

	token := strings.Fields(auth)
	if len(token) != 2 || strings.ToLower(token[0]) != "bearer" || token[1] == "" {
		return "", fmt.Errorf("authorization header invaild")
	}

	return token[1], nil
}`

Reply:
1. User should use name and password to login from browser.
2. Server validate the password, then generate a JWT token and set the token to cookie (why not header, the project has websocket service, it cannot set headers)
3. The next request will contains JWT token in cookie, so we donot set it in header again.

Hope it's useful for you.

Reply:
thanks，I get it ,your means is that Cookies cannot be disabled ?

Reply:
Yes, cookies set from server side, login will set cookie, logout will delete it.